### ISSUE TITLE
[bug] Can't delete custom emoji

### ISSUE THREAD

**Issue by chaoticryptidz** [superseriousbusiness/gotosocial#2136]
### Describe the bug with a clear and concise description of what the bug is.

Deleting custom emojis via web ui or API is currently broken.

### What's your GoToSocial Version?

0.10.0

### GoToSocial Arch

x86_64 nixos unstable

### What happened?

When you delete custom emojis and press the delete button, the API returns 500 Internal Server Error

### What you expected to happen?

The emoji to be deleted.

### How to reproduce it?

Create a custom emoji in the admin UI and then try to delete it.

### Anything else we need to know?

Log from server: 
```
Aug 18 23:32:41 social gotosocial[1302]: timestamp="18/08/2023 23:32:41.086" func=middleware.Logger.func1.1 level=ERROR latency=3.719839ms userAgent="Mozilla/5.0 (X11; Linux x86_64; rv:109.0) Gecko/20100101 Firefox/116.0" method=DELETE statusCode=500 path=/api/v1/admin/custom_emojis/01HGPP84WAW93VSBMBGNBX71P9 error="Error #01: EmojiDelete: db error: ERROR: malformed array literal: "01HGPP84WAW93VSBMBGNBX71P9" (SQLSTATE 22P02)\n" requestID=fdzwe2ma04000k0kn0fg msg="Internal Server Error: wrote 54B"
```

**Comment by tsmethurst** [comment]
Hmm I can't replicate this on 0.10.0 on my machine (linux amd64) with the testrig.

You didn't happen to update to 0.11.0 and then downgrade again did you, by any chance? :thinking: Just to eliminate things.

**Comment by chaoticryptidz** [comment]
started on 0.10.0, havent updated or rebuilt sinceseems to only sometimes happencan run db queries if might helpOn 19 Aug 2023, at 5:01 pm, tobi ***@***.***> wrote:﻿
Hmm I can't replicate this on 0.10.0 on my machine (linux amd64) with the testrig.
You didn't happen to update to 0.11.0 and then downgrade again did you, by any chance? 🤔 Just to eliminate things.

**Comment by chaoticryptidz** [comment]
fixed in 0.11.0
